overlayer/textractoverlayer/image_tools.py

```diff
@@ -5,14 +5,14 @@
 import boto3
 
 # Conditionally add /opt to the PYTHON PATH
-if os.getenv('AWS_EXECUTION_ENV') is not None:
```

Comment:
Can you leave the "" format? Same applies to the other ' to " changes. Just clutters the commit.